doc/source/web.rst

```diff
@@ -1,7 +1,44 @@
 .. _web:
 
-Web
-===
+Streaming data from cloud sources
+=================================
+
+Spatial Temporal Asset Catalogs
+-------------------------------
+
+`STAC <https://stacspec.org/en>`_
+
+Stream Sentinel-2 level 2A data from Element84
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    from pathlib import Path
+    from geowombat.core.stac import open_stac
+    from rasterio.enums import Resampling
+
+    # Bounds in lat/lon WGS84
+    bounds = (left, bottom, right, top)
+
+    data, df = open_stac(
+        stac_catalog='element84',
+        bounds=bounds,
+        epsg='epsg:8858',
+        nodata_fill=32768,
+        collection='sentinel_s2_l2a',
+        bands=['B02', 'B03', 'B04', 'B08'],
+        cloud_cover_perc=90,
```

Comment:
Assuming this is max cloud cover?

Reply:
Yeah, see [the docs](https://github.com/jgrss/geowombat/blob/dde2b87a2175fe72e064aa8a366677248f6bffd8/src/geowombat/core/stac.py#L172)